publish_job.py

```diff
@@ -62,21 +55,46 @@ def get_sts_and_sns_clients(aws_auth_method):
     return sts_client, client
 
 
+def find_output():
+    """
+    This function parses the stdout file and extracts the workflow output
+    :return:
+    """
+    # open stdout file
+    f = open("stdout.txt", "r")
+    stdout = f.read()
+
+    pattern = regex.compile(r'\{(?:[^{}]|(?R))*\}')
+    json_found = pattern.findall(stdout)
+    cwl_output = json.loads(json_found[-1])
```

Comment:
Does this line make job notifications dependent on a certain ordering of the workflow steps? For example, if we had a few more steps run between stage-out and publish-job, would the needed json blob still be in the same place?

Reply:
This code snippet scans the contents and pulls out the last JSON blob. So in terms of sequence, the assumption is that the last step would be the stage-out.